thirdeye/docs/contribute_datasource.rst

```diff
@@ -0,0 +1,42 @@
+..
+.. Licensed to the Apache Software Foundation (ASF) under one
+.. or more contributor license agreements.  See the NOTICE file
+.. distributed with this work for additional information
+.. regarding copyright ownership.  The ASF licenses this file
+.. to you under the Apache License, Version 2.0 (the
+.. "License"); you may not use this file except in compliance
+.. with the License.  You may obtain a copy of the License at
+..
+..   http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing,
+.. software distributed under the License is distributed on an
+.. "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+.. KIND, either express or implied.  See the License for the
+.. specific language governing permissions and limitations
+.. under the License.
+..
+
+.. _contribute-datasource:
+
+Didn't Find the Data Source You Want? Contribute!
+==================================================
+
+With the interfaces ThirdEye provide, it is not too difficult to add a new data source. 
```

Comment:
Can you add which interface to implement? Also with a short blurb on how to implement with examples?

Reply:
OK. Details added.